src/04-modality-specific-files/06-physiological-and-other-continous-recordings.md

```diff
@@ -22,17 +22,21 @@ directory. For example:`task-movie_stim.tsv.gz`
 Physiological recordings such as cardiac and respiratory signals and other
 continuous measures (such as parameters of a film or audio stimuli) can be
 specified using two files: a gzip compressed TSV file with data (without header
```

Comment:
requiring data to be in a gzip-compressed TSV file seems limiting, no? E.g. if I have a `wav` file with stimuli, converting that into TSV feels clunky to me...

Reply:
I don't know the historical context of this decision, but I think changing it is out-of-scope of this PR.